ModuleSystem/module_scripts_ai.py

```diff
@@ -2795,34 +2795,66 @@
 #White Mountains mountaintops, far apart, and update the values below with them
 tld_white_mountains_p1=( 26,-1)
 tld_white_mountains_p2=(-44,24)
+tld_white_mountains_min_y=-1200   # An Y*100 coordinate always North of the WM - I'm using Hornburg Y right now
 #And let me handle the rest obtaining the line equation y=mx+n
 tld_white_mountains_m=(tld_white_mountains_p1[1]-tld_white_mountains_p2[1])*1.0/(tld_white_mountains_p1[0]-tld_white_mountains_p2[0])
 tld_white_mountains_n=tld_white_mountains_p1[1]-tld_white_mountains_m*tld_white_mountains_p1[0]
 tld_white_mountains_m=int(round(tld_white_mountains_m*100,0))
 tld_white_mountains_n=int(round(tld_white_mountains_n*10000,0)) # When doing y=x*m + n, both x and m are mult * 100, k now must be mult an extra *100

```

Comment:
I think that the Middle-earth map was built upside down by @mtarini (he also coded that check). 

That's why @Merlkir had to flip the compass 180 degrees to make it look correct, even if the coordinate system is still internally messed up.

Let me know if you plan to improve the code legibility in following commits, I think we can merge this.